wpa_project/info/templates/info/rules.html

```diff
@@ -0,0 +1,150 @@
+        </li>
+        <li>
+            If present, the direction of the Range Master must be followed at all times. If a Range Master is on
+            duty, all users must clearly communicate arrow retrieval intervals with each other.
+        </li>
+        <li>
+            When the last person on the shooting line has finished and stepped behind the shooting line and the line
+            is clear, shout "WEST Range Clear" (if you are on the WEST Range) and "EAST Range Clear" (if you are on
+            the EAST Range).
+        </li>
+        <li>
+            Everyone retrieves arrows in UNISON for the Range you are on. No Exceptions
+        </li>
+        <li>
+            If you are the last on the shooting line, be courteous and don't make others wait for you to finish
+            shooting additional arrows. When retrieving arrows, do so in a timely manner so archers are not waiting
+            for your return to the shooting line before shooting can commence.
+        </li>
+        <li>
+            <strong>No sky draws.</strong> Do not raise the arrow point higher than the top of the bale when
+            drawing the bow.
+        </li>
+        <li>
+            Arrows are to be <strong>pointed down range</strong> at all times once the arrow is drawn from the quiver.
+        </li>
+        <li>
+            Arrows must be aimed and released at target butts only. Do not shoot across lanes. <strong>Never</strong>
+            point a loaded arrow at anything other than the target directly in front of you.
+        </li>
+        <li>
+            No placement of any temporary (portable) targets or target butts on the rage except directly in front of
+            permanent target butts. Any portable target must have its own stand and backstop.
+        </li>
+        <li>
+            Spectators must remain behind the shooting line at all times. Be respectful of archers' concentration
+            and do not attempt to distract them.
+        </li>
+        <li>
+            Shooting is permitted only from the designated shooting line. No one may shoot in front of or behind the
+            shooting line. (The organized Clout Round shooting near the west fence is the only exception.)
+        </li>
+        <li>
+            Archers under 18 years of age must be accompanied and directly supervised by an adult at all times while
+            on the range.
+        </li>
+        <li>
+            <strong>This range is for target arrows only. No hunting broadheads or practice broadheads of any kind allowed.
+            No exceptions.</strong>
+        </li>
+        <li>
+            No disruptive or loud behavior. Be considerate to all archers and spectators.
+        </li>
+        <li>
+            No unsolicited coaching or unauthorized instruction.
+        </li>
+    </ul>
+
+    <button class="btn btn-primary" onclick="goBack()">Go Back</button>
+
+    <script nonce="{{request.csp_nonce}}">
```

Comment:
Back button seems like it's not working for me. Not sure if order matters and the script needs to be above the button element?